src/toolkits/drawing_classifier/dc_data_iterator.hpp

```diff
@@ -58,9 +58,10 @@ class data_iterator {
      */
     std::vector<std::string> class_labels;
 
-    /**
-     * Whether to traverse the data more than once.
-     */
+    /** Whether this is training data or not. */
```

Comment:
this should be in a separate pr for dc_data_iterator. I have a dependency on it. I can hold my pr for prediction.

Reply:
Which part do you mean? Adding the `is_train` parameter?

Reply:
yeah, that's it.

Reply:
I think the plan now is to merge this pr asap. I need `create_data_iterator`. I will switch to save and load today.